### ISSUE TITLE
Does ZeroNet work in tails OS ?

### ISSUE THREAD

**Issue by Fr33M9nd** [HelloZeroNet/ZeroNet#498]
Hi , 
Tried to install ZeroNet in tails but wouldnt work. 
port failure and couldnt connect .... 

I followed the debian install instructions and everything seem to install fine 
then after typing python ZeroNet.py ( i think) it started going through the motions and doesnt launch 
with a bunch of connection problems in the terminal window...

Im thinking tails tor setup is conflicting somehow 

Is there a workaround for tails OS ?

Thanks 

Fr33M9nd

**Comment by HelloZeroNet** [comment]
yes: zeronet.py --proxy 127.0.0.1:9050 --tor disable --ui_ip 10.0.2.16 (your ip can be different from this use /sbin/ifconfig to see yours)
Then access using http://10.0.2.16:43110

**Comment by Fr33M9nd** [comment]
Hi thanks for the reply ....
This is where im at ....

an ifconfig or /sbin/ifconfig  shows inet addr:192.168.0.103

so did the following .....


python zeronet.py --proxy 127.0.0.1:9050 --tor disable --ui_ip 192.168.0.103
- Starting ZeroNet...
[16:28:58] - OpenSSL loaded, version: 0100010BF
[16:28:58] - Patching sockets to socks proxy: 127.0.0.1:9050
[16:28:58] - Version: 0.3.7 r1287, Python 2.7.9 (default, Mar  1 2015, 18:22:53) 
[GCC 4.9.2], Gevent: 1.0.1
[16:28:58] - Creating FileServer....
[16:28:58] - Creating UiServer....
[16:28:58] Site:1Name2..hM9F Content.json not exist: data/1Name2NXVi1RDPDgf5617UoW7xA6YrhM9F/content.json
[16:28:58] - Removing old SSL certs...
[16:28:58] - Starting servers....
[16:28:58] Ui.UiServer --------------------------------------
[16:28:58] Ui.UiServer Web interface: http://192.168.0.103:43110/
[16:28:58] Ui.UiServer --------------------------------------
[16:28:59] Site:1Name2..hM9F Content.json not exist: data/1Name2NXVi1RDPDgf5617UoW7xA6YrhM9F/content.json
[16:29:00] FileServer Checking port 15441 using portchecker.co...
[16:29:03] FileServer [BAD :(] Port closed: Port 15441 is closed.
[16:29:03] FileServer Trying to open port using UpnpPunch...
[16:29:04] FileServer UpnpPunch run error: error: [Errno 1] Operation not permitted in FileServer.py line 72 > UpnpPunch.py line 208 > UpnpPunch.py line 38 > socket.py line 474
[16:29:04] FileServer Upnp mapping failed :( Please forward port 15441 on your router to your ipaddress


any thoughts ?

Fr33M9nd

**Comment by HelloZeroNet** [comment]
its normal, the webui should be visible on http://192.168.0.103:43110/ if not:
- Go to address about:preferences#advanced
- Click Settings...
- Enter 192.168.0.103 to field No proxy for
